types/types.go

```diff
@@ -1,27 +1,34 @@
 package tests
 
 import (
+	"os"
+
 	"github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
 )
 
 var (
 
-	// Namespace where the chaos will be performed
+	// ChaosNamespace where the chaos will be performed
 	ChaosNamespace = "litmus"
-
-	// Namespace where the csp pods are deployed
+	// CspPodNs where the csp pods are deployed
 	CspPodNs = "openebs"
-
-	//Label of the CSP Pods
+	//CspPodLabels of the CSP Pods
 	CspPodLabels = "app=cstor-pool"
-
-	//Namespace where target pod is deployed
+	//TargetPodNs Namespace where target pod is deployed
 	TargetPodNs = "openebs"
-
-	//Label of the target pod
+	//TargetPodLabels Label of the target pod
 	TargetPodLabels = "openebs.io/target=cstor-target"
-
-	//Chaos Duration of the Experiment
+	//RepoName of the image to be used
+	RepoName = os.Getenv("REPO_NAME")
+	//RunnerImage name
+	RunnerImage = os.Getenv("RUNNER_IMAGE")
+	//OperatorImage name
+	OperatorImage = os.Getenv("OPERATOR_IMAGE")
+	//ExperimentImage name
+	ExperimentImage = os.Getenv("EXPERIMENT_IMAGE")
+	//ImageTag "latest or ci"
+	ImageTag = os.Getenv("IMAGE_TAG")
```

Comment:
What if we want a different image tag for all images?

Reply:
Can you make this feature optional, We can only do this if these envs are provided otherwise we will use the official image for all components.

Reply:
1. That is not the right situation, supporting common tags for all components.
2. The default value is already defined in the Gitlab ENVs no need to define it anywhere here.

Reply:
But the tags can be different. If we will support different tags for all components then it will be a good thing. I think these changes are also straight forward.

Reply:
It can be done.
But it is meaningless to test (say) operator/runner with ci and experiment with the latest.
I think we discussed this @ksatchit during the last release discussion.

Reply:
Even though if you guys feel it should be there I can add this.

Reply:
I will tell you a case:
- I want to use ci image of runner/operator but for ansible-runner I want to use `shubh214/asnible-runner:test` then how we will achieve this?

Reply:
For this case the runner/operator image should be -- `shubh214/<image-name>:ci` (as we are not providing different repo names it is future scope if feel the need to add this then)
and you have to make/change the ansible-runner image tag to `ci` only.
If we have suppose 3-4 components ideally we should not have to maintain 3-4 image tags for each components and asking user to have same component.

Reply:
I don't get your requirements. Can we sync up once? tell me whenever you will be free.?

Reply:
will look into it